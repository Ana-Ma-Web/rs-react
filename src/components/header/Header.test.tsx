// import React from 'react';
// import { cleanup, render, screen } from '@testing-library/react';
// import Header from './Header';

it('should have anime', () => {
  // render(<Header />);
  // const message = screen.queryByText(/anime/i);
  // // const details = screen.queryByTestId('details');
  // // if (!details) {
  // // } else expect(message).toBeNull();
  // expect(message).toBeVisible();
  // afterEach(cleanup);
});

it('should match error button', () => {
  // // screen.debug();
  // render(<Header />);
  // const errorBtn = screen.queryByText(/error button/i);
  // expect(errorBtn).toMatchSnapshot();
  // afterEach(cleanup);
});
