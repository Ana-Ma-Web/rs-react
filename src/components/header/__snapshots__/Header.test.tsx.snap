// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should have anime 1`] = `
<button
  class="btn_error"
>
  ERROR BUTTON
</button>
`;

exports[`should match error button 1`] = `
<button
  class="btn_error"
>
  ERROR BUTTON
</button>
`;
